doc/nrf/known_issues.rst

```diff
@@ -489,6 +489,14 @@ KRKNWK-6073: Potential delay during FOTA
 Matter (Project CHIP)
 =====================
 
+.. rst-class:: v1-7-0
```

Comment:
```suggestion
.. rst-class:: v1-6-1
```
Please do not add tag for next release before it took place. The current version is still 1.6.1, so please use `v1-6-1` or wait till after RC1 and the PR that adds 1.7.0.